actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserFeedActor.java

```diff
@@ -40,6 +40,9 @@ public void onReceive(Request request) throws Throwable {
     } else if (ActorOperations.CREATE_USER_FEED.getValue().equalsIgnoreCase(operation)) {
       logger.info(context, "UserFeedActor:onReceive createUserFeed method called");
       createUserFeed(request, context);
+    } else if (ActorOperations.DELETE_USER_FEED.getValue().equalsIgnoreCase(operation)) {
```

Comment:
Now this Actor handling multiple message so better use case statement instead of if , else if clause.